en/publish/publish-oppo-instant-games.md

```diff
@@ -138,6 +132,12 @@ Then switch to the **Package Load** section of **Mini Game Debugger**, click Ref
 
 **Note**: Subpackage rpk needs to be copied to the `/sdcard/subPkg/` directory of Android devices, and non-subpackage rpk needs to be copied to the `/sdcard/games/` directory of Android devices, both of which cannot be mixed.
 
+## Resource Management for OPPO Mini Game Environment
+
+**OPPO Mini Game** is similar to **WeChat Mini Game**. There are restrictions on the package size. Resources over **10MB** must be downloaded via a network request.
```

Comment:
Resources over **10MB** must be downloaded via a network request. -> The main package size limit for OPPO Mini Game is **10MB**, more than that must be downloaded via a network request.